Assets/MixedRealityToolkit.Services/InputSimulation/Implementation/InputSimulationService.cs

```diff
@@ -0,0 +1,430 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Utility struct that provides mouse delta in pixels (screen space), normalized viewport coordinates, and world units.
+    /// </summary>
+    public class MouseDelta
+    {
+        public Vector3 screenDelta = Vector3.zero;
+        public Vector3 viewportDelta = Vector3.zero;
+        public Vector3 worldDelta = Vector3.zero;
+    }
+
+    [MixedRealityDataProvider(
+        typeof(IMixedRealityInputSystem),
+        SupportedPlatforms.WindowsEditor | SupportedPlatforms.MacEditor | SupportedPlatforms.LinuxEditor,
+        "Input Simulation Service",
+        "Profiles/DefaultMixedRealityInputSimulationProfile.asset",
+        "MixedRealityToolkit.SDK")]
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/InputSimulation/InputSimulationService.html")]
+    public class InputSimulationService :
```

Comment:
If there's any way to preserve the file history here that would be great -this is showing up as a pure add/delete (meaning you lose history at this change). Was trying to review this against the old (and it's hard because it shows up as a pure add)

Reply:
I can move it back into the Editor folder for this PR, and then make a pure file move commit later.

The purpose of moving these files is to keep the service implementation (which happens to be editor-only atm) separate from actual inspectors etc.. That will also make it easier to change assembly settings if we ever figure out how to solve the input sim profile warnings in standalone builds #3971